README.md

```diff
@@ -1,15 +1,34 @@
 # Lokalise - the unofficial node lokali.se client.
 
-This is all still very WIP.
```

Comment:
Although I am open to change this text, I would still like a disclaimer that this is still not recommended for production (although it's been battle tested for some time). Let me think of something else a bit later. For now let's leave this?

Reply:
👍